validate.go

```diff
@@ -89,17 +93,17 @@ func bundleValidate(spec rspec.Spec, rootfs string) {
 func checkSemVer(version string) {
 	re, _ := regexp.Compile("^(\\d+)?\\.(\\d+)?\\.(\\d+)?$")
 	if ok := re.Match([]byte(version)); !ok {
-		logrus.Fatalf("%s is not a valid version format, please read 'SemVer v2.0.0'", version)
+		logrus.Fatalf("%q is not a valid version format, please read 'SemVer v2.0.0'", version)
 	}
 }
 
 func checkMounts(mounts []rspec.Mount, rootfs string) {
 	for _, mount := range mounts {
 		rootfsPath := path.Join(rootfs, mount.Destination)
 		if fi, err := os.Stat(rootfsPath); err != nil {
-			logrus.Fatalf("Cannot find the mount point: %v", rootfsPath)
+			logrus.Fatalf("Cannot find the mount destination %q", rootfsPath)
 		} else if !fi.IsDir() {
-			logrus.Fatalf("Mount point: %v is not a directory.", rootfsPath)
+			logrus.Fatalf("Mount destination %q is not a directory.", rootfsPath)
```

Comment:
It predates this PR, but `rootfsPath` is a strange name for a variable holding the mount destination.